### PR TITLE
fix instructions to use blogdown

### LINKED ISSUE #194: How to render R Markdown posts
Question by @steffilazerte https://github.com/ropensci/roweb3/pull/364

### PR DESCRIPTION
**Pull request by maelle**
Fix #194

### CODE REVIEW

**Comment by maelle** [comment]
@steffilazerte friendly reminder about this PR :smile_cat:

File: authortechnical.Rmd [hunk 1]
```diff
@@ -24,6 +24,14 @@ This chapter links to templates for posts and checklists that you can also find
 Fork the rOpenSci website [repository](https://github.com/ropensci/roweb3) and create a new branch to work on your post. 
 For help with this aspect of git/GitHub, we recommend [happygitwithr](https://happygitwithr.com/fork-and-clone.html) and the [pull request helpers of the usethis package](https://usethis.r-lib.org/articles/articles/pr-functions.html).
 
+## Install blogdown
+
+If you plan to use R Markdown to write your post (as opposed to directly writing it in Markdown, not executing any code), please install or update blogdown.
```
Comment (steffilazerte) [inline authortechnical.Rmd:29]:
```suggestion
If you plan to use R Markdown to write your post (using `index.Rmd` as opposed to directly writing it in Markdown with `index.md` and not executing any code), please install or update blogdown.
```

**Review by steffilazerte — COMMENTED** [review]
Eeep, sorry, I forgot, thanks for the nudge. Yes looks good! One suggestion (to be super clear about the file types).

**Comment by maelle** [comment]
Thank you!!

**Pull request merged**